examples/MauiApp2/MauiApp2.csproj

```diff
@@ -1,8 +1,11 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
 	<PropertyGroup>
-		<TargetFrameworks>net6.0-android;net6.0-ios;net6.0-maccatalyst</TargetFrameworks>
-		<TargetFrameworks Condition="$([MSBuild]::IsOSPlatform('windows'))">$(TargetFrameworks);net6.0-windows10.0.19041.0</TargetFrameworks>
+		<!-- <TargetFrameworks>net7.0-android;net7.0-ios;net7.0-maccatalyst</TargetFrameworks> -->
+		<TargetFrameworks>net7.0-android</TargetFrameworks>
```

Comment:
Would still like the build-server to verify that the MAUI-App can be compiled for all major platforms (net7.0-android;net7.0-ios;net7.0-maccatalyst)

Reply:
Setting it up like that does not work because of mac requires a mac to compile. it's easiest to just set the TargetFrameworks through appveyor